tests/tracking/_model_registry/test_utils.py

```diff
@@ -10,6 +10,11 @@
 from mlflow.tracking._model_registry.utils import _get_store
 from mlflow.tracking._tracking_service.utils import _TRACKING_URI_ENV_VAR
 
+# Disable mocking tracking URI here, as we want to test setting the tracking URI via
```

Comment:
Just added an explanatory comment for cases where we do want to use `notrackingurimock`